core/org.wso2.carbon.base/src/test/java/org/wso2/carbon/base/CarbonBaseUtilsTest.java

```diff
@@ -50,7 +50,7 @@ public void setSecurityManager() {
             String resourcePath = resourceURL.getPath();
             resourcePath = resourcePath + "policy-test.policy";
             System.setProperty("java.security.policy", resourcePath);
-            System.setSecurityManager(new SecurityManager());
+//            System.setSecurityManager(new SecurityManager());
```

Comment:
Security manager not working properly with Java 17. Hence need to get rid of security manager.